Objective-C/CBLFullTextIndexConfiguration.h

```diff
@@ -41,11 +41,25 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly, nullable) NSString* language;
 
 /**
- Constructor for creating a full-text index by using an array of expression strings
+ A predicate expression defining conditions for indexing documents.
+ Only documents satisfying the predicate are included, enabling partial indexes.
+ */
+@property (nonatomic, readonly, nullable) NSString* where;
+
+/**
+ Initializes a partial full-text index by using an array of expression strings
+ */
+- (instancetype) initWithExpression: (NSArray<NSString*>*)expressions
```

Comment:
Sorry, I might have suggested to put the word partial there in the comment. It is not correct here as there is no where parameter.

Reply:
that was a typo, will fix